objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
Of course we've discussed 3D lists, or 4359D lists. That's what formats like JSON or XML are for, as you say, and that's what the **first instance of this PR** did. JSON and XML can represent *any kind of list* that we can have in Snap<i>!</i>, and that's why they're absolutely uncontroversial.

That's also why I didn't add CSV or plain text in the beginning, and you've all definitely proved my point that I should have stayed out of it, and let somebody else add support for CSV and plain text after (if) this PR went through, and let that somebody else take the bashing over commas, newlines and dimensions too.